Week2/README.md

```diff
@@ -45,10 +45,10 @@ To define a Primary Key while creating the table, you should determine the attri
 ```sql
 CREATE TABLE teachers (
       teacher_number INT,
-      teacher_name VARCHAR(50),
+      name VARCHAR(50),
       date_of_birth DATE,
       subject TEXT,
-      gender ENUM('m', 'f'),
+      email VARCHAR(200),
```

Comment:
am not sure why ENUM type needs to removed ?
It was added so that they know and practice different data types in columns

Reply:
Not any particular reason except that the field didn't seem to fulfill any purpose in the subjects of this week.
I added the `email` field because it's a good example for the `UNIQUE` constraint and the `gender` field didn't seem important.
But now that we are talking about it; I would never suggest a student to use `ENUM`. It's a bad practice IMHO. Having a separate table with lookup values is almost always better because you can query those values and edit those values easily.

Reply:
The purpose was to introduce possible data types in SQL.  They can learn with experience which are good and bad (VARCHAR, TEXT etc. would also have similar arguments for and against)

Reply:
I was about to restore the ENUM but then it occurred to me; that is content for week 1, right?
https://github.com/HackYourFuture/databases/tree/master/Week1#6-what-are-data-types-as-applied-to-databases
And because it doesn't serve of explaining week 2 concepts I could remove it.

Reply:
You are right. I won't fight about it ;)
ENUM can be removed from this example.